detections/application/splunk_command_and_scripting_interpreter_delete_usage.yml

```diff
@@ -0,0 +1,62 @@
+name: Splunk Command and Scripting Interpreter Delete Usage
+id: 8d3d5d5e-ca43-42be-aa1f-bc64375f6b04
+version: 1
+date: '2022-05-27'
+author: Michael Haag, Splunk
+type: Anomaly
+datamodel:
+- Splunk_Audit
+description: The following analytic identifies the use of the risky command - Delete - that may be utilized in Splunk to delete some or all data queried for. In order to use Delete in Splunk, one must be assigned the role. This is typically not used and should generate an anomaly if it is used.
+search: '| tstats `security_content_summariesonly` count min(_time) as firstTime max(_time) as lastTime from datamodel=Splunk_Audit.Search_Activity where Search_Activity.search IN ("*| delete*") Search_Activity.search_type=adhoc Search_Activity.user!=splunk-system-user
+  by Search_Activity.search Search_Activity.info Search_Activity.total_run_time Search_Activity.user Search_Activity.search_type 
+  | `drop_dm_object_name(Splunk_Audit)` 
+  | `security_content_ctime(firstTime)` 
+  | `security_content_ctime(lastTime)` 
+  | `splunk_command_and_scripting_interpreter_delete_usage_filter`'
+how_to_implement: To successfully implement this search acceleration is recommended against the Search_Activity datamodel that runs against the splunk _audit index. 
```

Comment:
I would include a line that this search requires the Common Information Model app which includes the Splunk Audit Datamodel found here: https://splunkbase.splunk.com/app/1621/